libs/external-router/guides/configuration.md

```diff
@@ -0,0 +1,22 @@
+# `@daffodil/external-router` Configuration
+
+The External Router exposes a configuration token that you can provide in order to configure your the behavior of the package.
```

Comment:
I think `The External Router exposes a configuration token you can provide to configure the behavior of the package` is better. 4 words removed (one was a typo, I think)

Reply:
https://github.com/graycoreio/daffodil/pull/1240/commits/e84f901398d30c4af371fb257714725339f3640a